packages/react-router-website/modules/examples/PreventingUnload.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import {
+  BrowserRouter as Router,
+  Route,
+  Link,
+  Prompt,
+  BrowserPrompt
+} from 'react-router-dom'
+
+const PreventingTransitionsExample = () => (
```

Comment:
If you click to navigate to google, it does it in the core browser, not the fake browser. So this event does fire. I would not release code without fully testing 😄 
Shall I take this out of the docs completely?